packages/create-nuxt3-app/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "create-nuxt3-app",
+  "engines": {
+    "node": ">=16.14.2",
+    "npm": ">=8.5.0"
+  },
+  "version": "0.0.1-alpha.4",
+  "description": "Create a nuxt 3 app in seconds",
+  "bin": "dist/index.js",
+  "scripts": {
+    "dev": "npm run build && npm i -g . && create-nuxt3-app",
+    "test": "vitest --run",
+    "lint": "eslint \"./src/**/*.{ts,tsx}\" --max-warnings=0",
+    "type": "tsc --noEmit",
+    "build": "rm -rf dist && tsup src/ --format cjs,esm --dts --dts-resolve",
+    "docs": "npx typedoc src"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
+    }
+  },
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "templates"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sidebase/create-nuxt3-app"
+  },
+  "keywords": [
+    "nuxt",
+    "vue",
+    "cli",
+    "create-nuxt-app",
+    "setup",
+    "typescript"
+  ],
+  "author": "sidebase <sidebase@sidestream.tech>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/sidebase/sidebase/issues/new/choose"
+  },
+  "homepage": "https://sidebase.io",
+  "devDependencies": {
+    "@types/fs-extra": "^9.0.13",
+    "@types/prompts": "^2.0.14",
+    "@typescript-eslint/eslint-plugin": "^5.36.2",
+    "@typescript-eslint/parser": "^5.36.2",
+    "@vitest/coverage-c8": "^0.23.1",
+    "eslint": "^8.23.0",
+    "tsup": "^6.2.3",
+    "typescript": "^4.8.2",
+    "vitest": "^0.23.1"
+  },
+  "dependencies": {
+    "chalk": "^5.1.2",
+    "defu": "^6.1.0",
+    "fs-extra": "^10.1.0",
+    "node-plop": "^0.31.0",
+    "prompts": "^2.4.2"
```

Comment:
please check if all of these are still actually used in the final package (:

Reply:
I do use all of them, however I (generally still) often wonder if I really need them as dependencies or if it is enough to include them as dev-dependencies.

On all of their installation pages they were being added as non dev-dependencies. Does that mean they need to be added regularly or can I check it another way?